Clipboard.xs

```diff
@@ -528,6 +556,47 @@ PPCODE:
 	XSRETURN_NO;
     }
 
+void
+SetBitmap(dib)
+    SV *dib
+PPCODE:
+    HANDLE myhandle;
+    HGLOBAL hGlobal;
+    STRLEN leng;
+    U8 *str = (U8*) SvPV(dib, leng);
+
+    if ( hGlobal = GlobalAlloc(GMEM_DDESHARE, (leng+2)*sizeof(char)*2) ) {
+        wchar_t *szString = (wchar_t *) GlobalLock(hGlobal);
+	U8 * const send = str + leng;
+	/* we have raw data */
+	while (str < send) {
```

Comment:
This expansion from bytes to wchar_t looks wrong, [Standard Clipboard Formats](https://docs.microsoft.com/en-us/windows/win32/dataxchg/standard-clipboard-formats) claims it's a

> A memory object containing a BITMAPINFO structure followed by the bitmap bits.

and doesn't mention converting it like this.